common/src/main/java/com/revenuecat/purchases/common/logWrapper.kt

```diff
@@ -2,62 +2,69 @@ package com.revenuecat.purchases.common
 
 import com.revenuecat.purchases.strings.Emojis
 
+const val V2_LOGS_ENABLED = false
+
 fun log(intent: LogIntent, message: String) {
-    val emojifiedMessage = "${intent.emojiList.joinToString("")} $message"
+    var fullMessage = ""
+    if (V2_LOGS_ENABLED) {
+        fullMessage = "${intent.emojiList.joinToString("")} $message"
+    } else {
+        fullMessage = message
+    }
```

Comment:
This could be written like:

```
val fullMessage  = if (V2_LOGS_ENABLED) {
         fullMessage = "${intent.emojiList.joinToString("")} $message"
     } else {
         fullMessage = message
     }
```

That way, `fullMessage` is immutable 👓